apps/transport/client/javascripts/map.js

```diff
@@ -14,6 +14,7 @@ const Mapbox = {
 
 const regionsUrl = '/api/stats/regions'
 const aomsUrl = '/api/stats/'
+const bikesUrl = '/api/stats/bikes'
```

Comment:
maybe `bike-sharing` to be consistent with the rest?

Reply:
ok!